### PR TITLE
fix(de): View Type Dropdown will use the defined types

### PR DESCRIPTION
**Pull request by Palakp41**
- Moved the visualization types and loading of those files from flows to shared folder
- Updated the View type dropdown in data explorer to use the defined types in shared folder

### CODE REVIEW

File: src/timeMachine/components/visualization/ViewTypeDropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import React, {FC} from 'react'
+import {TYPE_DEFINITIONS} from 'src/shared/visualization'
+
+// Components
+import {Dropdown, DropdownMenuTheme} from '@influxdata/clockface'
+import {ComponentStatus} from 'src/clockface'
+
+// Utils
+import {isFlagEnabled} from 'src/shared/utils/featureFlag'
```
Comment (drdelambre) [inline src/timeMachine/components/visualization/ViewTypeDropdown.tsx:9]: why is this file here instead of using the `src/shared/visualization` version?

File: src/timeMachine/components/visualization/view.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+// Libraries
+import React, {FC, useCallback} from 'react'
+import {useDispatch, useSelector} from 'react-redux'
+
+// Components
+import ViewTypeDropdown from 'src/timeMachine/components/visualization/ViewTypeDropdown'
```
Comment (drdelambre) [inline src/timeMachine/components/visualization/view.tsx:6]: import this component from `src/shared/visualization` instead

File: src/timeMachine/components/visualization/ViewTypeDropdown.tsx (hunk 1 above)
Comment (Palakp41) [reply src/timeMachine/components/visualization/ViewTypeDropdown.tsx]: Good catch, thanks!

**Review by drdelambre — APPROVED** [review]
:crossed_fingers: i think it looks good
